docs/developers/analyzer-development.md

```diff
@@ -2,119 +2,342 @@
 hide:
   - footer
 ---
-# Write analyzers in Timesketch
+# Write analyzers for Timesketch
 
-## Multi Analyzer
+## Overview
 
-When you develop an analyzer that would benefit from creating smaller sub-jobs, you should use Multi Analyzer.
```

Comment:
as far as I can tell your update would remove the whole info about multi vs non multi analyzer, is that by purpose?